static/main.js

```diff
@@ -30,15 +30,15 @@ function getResult() {
 			var data={'formula':enteredFormula};
 			var url = "";
 			
-			if (enteredFormula.match("^wh") || enteredFormula.match("^Wh") || lang == "hn") {
+			if (enteredFormula.indexOf("=") >= 0) {
```

Comment:
Is the rationality if the query contains a = the program behaves like a calculator? Maybe that should be stated somewhere.